lib/ui/food_detail.dart

```diff
@@ -5,88 +5,88 @@ import 'package:font_awesome_flutter/font_awesome_flutter.dart';
 import 'package:google_fonts/google_fonts.dart';
 
 class FoodDetail extends StatelessWidget {
-
   final colorRed = Colors.redAccent.withOpacity(0.7);
```

Comment:
if you are tryna make better name convention user _varName format in class valiables that are supposed to be private. same for functions i wont comment same thing anywhere else